packages/dina-ui/components/seqdb/sanger-workflow/SangerPcrReactionStep.tsx

```diff
@@ -0,0 +1,250 @@
+    setEditMode(false);
+  }
+
+  const PCR_REACTION_COLUMN: Column<PcrBatchItem>[] = [
+    {
+      Cell: ({ original }) => {
+        if (original?.wellRow === null || original?.wellColumn === null)
+          return <></>;
+
+        return <>{original.wellRow + "" + original.wellColumn}</>;
+      },
+      Header: <FieldHeader name={"wellCoordinates"} />,
+      sortable: false
+    },
+    {
+      Cell: ({ original }) => {
+        if (original?.cellNumber === undefined) return <></>;
+
+        return <>{original.cellNumber}</>;
+      },
+      id: "tubeNumber",
+      Header: <FieldHeader name={"tubeNumber"} />,
+      sortable: false
+    },
+    {
+      Cell: ({ original }) => {
+        const fetchedMaterialSample = materialSamples.find(
+          (materialSample) => materialSample.id === original?.materialSample?.id
+        );
+
+        if (!fetchedMaterialSample) return <></>;
+
+        return <p>{fetchedMaterialSample.materialSampleName}</p>;
+      },
+      Header: <FieldHeader name={"materialSampleName"} />,
+      sortable: false
+    },
+    {
+      Cell: ({ original }) => {
+        const fetchedMaterialSample = materialSamples.find(
+          (materialSample) => materialSample.id === original?.materialSample?.id
+        );
+
+        if (!fetchedMaterialSample) return <></>;
+
+        return <>{getScientificNames(fetchedMaterialSample)}</>;
+      },
+      Header: <FieldHeader name={"scientificName"} />,
+      sortable: false
+    },
+    {
+      Cell: ({ original }) => {
+        return editMode ? (
+          <div>
+            <AutoSuggestTextField
+              name={"results[" + original?.id + "]"}
+              hideLabel={true}
+              removeBottomMargin={true}
+              customOptions={(searchTerm, _) => {
+                return Object.values(PcrBatchItemDropdownResults).filter(
+                  (resultString) =>
+                    searchTerm
+                      ? resultString
+                          .toLowerCase()
+                          .startsWith(searchTerm.toLowerCase())
+                      : true
+                );
+              }}
+              blankSearchBackend={"json-api"}
+            />
+          </div>
+        ) : (
+          <div
+            style={{
+              backgroundColor: "#" + pcrBatchItemResultColor(original?.result),
+              borderRadius: "5px",
+              paddingLeft: "5px"
+            }}
+          >
+            {original?.result ?? ""}
+          </div>
+        );
+      },
+      Header: <FieldHeader name={"result"} />,
+      sortable: false
+    }
+  ];
+
+  // Wait until the PcrBatchItems have been loaded in before displaying the table.
+  if (loading) {
+    return <LoadingSpinner loading={true} />;
+  }
+
+  // Load the result based on the API request with the pcr-batch-item.
+  const initialValues = {
+    results: Object.fromEntries(
+      selectedResources.map((obj) => [obj.id, obj.result])
+    )
+  };
```

Comment:
This does not seem to be working correctly to populate the dropdown menus when going into edit mode. The dropdown menus should be pre-populated with it's previous value.